CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.1.1...master
 
+- Moved the legend on the distribution chart to the side
```

Comment:
We can link to https://ecosystem.atlassian.net/browse/JPERF-421, so that users can see the broader context and comment on the feature.
You can see how you can do it in the changelog entries below.

Reply:
Done